networking/hardware_networks/uninstalling-sriov-operator.adoc

```diff
@@ -0,0 +1,50 @@
+[id="uninstalling-sriov-operator"]
+= Uninstalling the SR-IOV Network Operator
+include::modules/common-attributes.adoc[]
+:context: uninstalling-sr-iov-operator
+
+toc::[]
+
+To uninstall the SR-IOV Network Operator, you must first delete any running SR-IOV workloads. You must also delete the webhooks after uninstalling the operator.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster using an account with `cluster-admin` permissions.
+* You have the SR-IOV Network Operator installed.
+
+.Procedure
+
+. Delete all SR-IOV custom resources (CRs).
+
+. Remove the SR-IOV Network Operator from your cluster by following the documentation on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+. The SR-IOV resource definitions (CRDs) remain in the cluster after the SR-IOV Network Operator is uninstalled. Delete the remaining SR-IOV CRDs:
++
+[source,terminal]
+----
+$ oc get crd -o name | grep sriov | xargs oc delete
+----
+
+. Delete the SR-IOV webhooks:
++
+[source,terminal]
+----
+$ oc delete mutatingwebhookconfigurations network-resources-injector-config
+$ oc delete MutatingWebhookConfiguration sriov-operator-webhook-config
+$ oc delete ValidatingWebhookConfiguration sriov-operator-webhook-config
+----
+
+. Delete references to stray finalizers from the SriovNetworks:
++
+[source,terminal]
+----
+$ oc get SriovNetwork -A -o name | sed 's:.*/::' | xargs oc patch SriovNetwork -n openshift-sriov-network-operator  -p '{"metadata":{"finalizers":null}}' --type=merge
+----
+
+. Delete the SR-IOV Network Operator namespace:
++
+[source,terminal]
+----
+$ oc delete namespace openshift-sriov-network-operator
+----
+
```

Comment:
Suggestion: is there a way to check this task has been successfully accomplished.

Reply:
It's better to include a confirmation step, if possible - we're often asked for them, and their use is well-defined in the mod docs guidelines. Validating that the actions were successful is never docs bloat. The sample syntax for the verification section is in [this template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc).